packages/backend/src/index.ts

```diff
@@ -81,6 +83,29 @@ async function main() {
       `Do NOT deploy this to production. Read more here https://backstage.io/docs/getting-started/`,
   );
 
+  // loadAllConfigSchema() from config-loader
+  // does similar things as backstage-cli config:schema
+  // optionally, pass schema in loadBackendConfig
+  // loadBackendConfig can optionally validate loaded config the schema
+  // loadBackendConfig tells the logger about secrets, so that it can filter out.
+
+  // get all packages
+  const schema = await loadConfigSchema({
+    dependencies: ['@backstage/integration'],
+  });
+  console.log(schema);
+  console.log('serialize');
+
+  const merged = mergeConfigSchemas(
```

Comment:
No need to merge the schemas afaik, this is just if you want to have a nice to display JSONSchema blob that isn't a bunch of separate small schemas

Reply:
Right, we need to get all the keys with visibility "secret" and so we thought merging is the first step. But we can obviously loop over all the individual schemas and do that. 😅

Reply:
All of that logic is already in the schema though, and you're not really interested in the keys at all, only the values. Something like `schema.process(config, {visibility: ['secret']})` should give you back a nested object with all secret values